config/defaults_template.yaml

```diff
@@ -27,7 +27,7 @@ languages:
   runImage: "{{ .RunImagePrefix }}cxx:{{ .Version }}"
 
 - language: go
-  buildImage: golang:1.20
+  buildImage: golang:1.22
```

Comment:
We can bump it to 1.23 so that we don't don't need to do this again when grpc-go drops support for 1.22.

Reply:
Or we could use `golang:latest`.

Reply:
Done.